### PR TITLE
Add Python 3.9/10. Cache pip test environment

### PR DESCRIPTION
**Pull request by fAndreuzzi**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 EXTRAS = {
     'docs': ['Sphinx==1.4', 'sphinx_rtd_theme'],
-    'test': ['pytest', 'pytest-cov', 'ezyrb'],
+    'test': ['pytest', 'pytest-cov', 'ezyrb>=1.3.0.post2301'],
```
[the remark this answers is not in the document]
Comment (fAndreuzzi) [reply setup.py:22]: We don't want ezyrb to bring in `GPy` :)

[the remark this answers is not in the document]
Comment (fAndreuzzi) [reply setup.py]: In 4cb40a0 I reduced the requirement to the minimum @ndem0

[the remark this answers is not in the document]
Comment (ndem0) [reply setup.py]: Yes, you're right!

**Comment by fAndreuzzi** [comment]
Let me know if I can merge this @ndem0 @mtezzele

**Pull request closed without merging**